### PR TITLE
⭐ Trigger mondoo-operator integration tests after pushing edge containers

### PR DESCRIPTION
**Pull request by imilchev**
- [x] Add a step to trigger the mondoo-operator integration tests with the new edge container
- [x] Add github token as a secret `GITHUB_API_TOKEN`

This can only be merged after we have the secret configured for this repo

Signed-off-by: Ivan Milchev <ivan@mondoo.com>

### CODE REVIEW

**Comment by benr** [comment]
Secret added.  I've renamed the variable because GITHUB_ is reserved.

**Comment by imilchev** [comment]
This is ready for merging. You can see it successfully running [here](https://github.com/mondoohq/client/actions/runs/2806405895) and it triggered the integration tests for the operator resulting in [this workflow](https://github.com/mondoohq/mondoo-operator/runs/7699353238?check_suite_focus=true)

File: .github/workflows/edge.yml [hunk 1]
```diff
@@ -59,6 +58,14 @@ jobs:
             VERSION=${{ github.event.inputs.version }}
           target: rootless
           tags: |
-            mondoo/client:edge-${{ github.event.inputs.version }}-rootless
             mondoo/client:edge-${{ github.event.inputs.version }}-rootless
             mondoo/client:edge-latest-rootless
+
+      - name: Start mondoo-operator integration tests
+        run: |
+          curl \
```
Comment (czunker) [inline .github/workflows/edge.yml:66]: nit: add `-s` , so we do not see this:
```
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
0    74    0     0  100    74      0    379 --:--:-- --:--:-- --:--:--   379
```

**Review by czunker — APPROVED** [review]
